pkg/tuned/controller.go

```diff
@@ -886,6 +886,54 @@ func getNodeName() string {
 	return name
 }
 
+// getCpuCount returns the total number of CPUs on the host as reported by the "lscpu" command or
+// 0 if an error is encountered.  Note that runtime.NumCPU() returns only "online" CPUs, hence the
+// need for this function.
+func getCpuCount() int {
```

Comment:
Let me try that.  That sounds cleaner than actually parsing the `lscpu` output, thank you!

Reply:
Right, I see what you mean now. This would actually require querying the API server or watching the nodes, none of which I'm particularly happy to do wrt to reliability/performance.  I believe the current implementation is way more reliable.  Do you have a specific reason in mind for suggesting that?